service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/StandardDirectionOrderIssuedEmailContentProviderTest.java

```diff
@@ -3,26 +3,41 @@
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
 import uk.gov.hmcts.reform.fpl.model.notify.allocatedjudge.AllocatedJudgeTemplateForSDO;
+import uk.gov.hmcts.reform.fpl.model.notify.sdo.CTSCTemplateForSDO;
+import uk.gov.hmcts.reform.fpl.service.config.LookupTestConfig;
+
+import java.util.List;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.populatedCaseDetails;
 
-@ContextConfiguration(classes = {StandardDirectionOrderIssuedEmailContentProvider.class})
+@ContextConfiguration(classes = {StandardDirectionOrderIssuedEmailContentProvider.class, LookupTestConfig.class})
+@TestPropertySource(properties = {"manage-case.ui.base.url=http://fake-url"})
 class StandardDirectionOrderIssuedEmailContentProviderTest extends AbstractEmailContentProviderTest {
 
     @Autowired
     private StandardDirectionOrderIssuedEmailContentProvider standardDirectionOrderIssuedEmailContentProvider;
 
     @Test
-    void shouldReturnExpectedMapWithValidSDODetails() {
+    void shouldReturnExpectedMapForJudgeWithValidSDODetails() {
         AllocatedJudgeTemplateForSDO expectedMap = allocatedJudgeSDOTemplateParameters();
 
         assertThat(standardDirectionOrderIssuedEmailContentProvider
             .buildNotificationParametersForAllocatedJudge(populatedCaseDetails()))
             .isEqualToComparingFieldByField(expectedMap);
     }
 
+    @Test
+    void shouldReturnExpectedMapForCTSCWithValidSDODetails() {
```

Comment:
should we also test the event in where the first hearing is not available thus an exception is thrown?

Reply:
its technically never possible for that to happen in the system so I don't think it needs testing, I throw the exception because I reused a method that returns <Optional> but for this usage there will always be a hearing